generic3g/tests/Test_VariableSpec_private.pf

```diff
@@ -0,0 +1,81 @@
+#include "MAPL_TestErr.h"
+module Test_VariableSpec_private
+   use mapl3g_VariableSpec_private
+   use esmf
+   use pfunit
+   use ESMF_TestMethod_mod
+   use mapl3g_EsmfRegridder, only: EsmfRegridderParam
+   implicit none(type,external)
+
+   character(len=*), parameter :: NONZERO = 'Non-zero status returned'
+   character(len=*), parameter :: EXCEPT_MSG = 'Invalid value'
+
+contains
+
+   @Test
+   subroutine test_validate_short_name()
+      integer :: status
+      
+      call validate_short_name('F00', rc=status)
+      @assertTrue(status == 0, NONZERO)
+
+      call validate_short_name('0F00', rc=status)
+      @assertExceptionRaised()
```

Comment:
You should consider using the form of `@assertExceptionRaised()` where you specify the expected error message.   A bit tedious until we are certain of the desired messages.